src/main/java/com/example/java/test/junior/developer/model/Question.java

```diff
@@ -0,0 +1,28 @@
+package com.example.java.test.junior.developer.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor(force = true)
+@AllArgsConstructor
+@Table(name = "questions")
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "question_id", nullable = false)
+    private int id;
+
+    @Column(name = "question", nullable = false)
+    public String name;
+
+    @ManyToOne(cascade = CascadeType.DETACH)
+    @JoinColumn(name = "category_id")
+    private Question question;
```

Comment:
Probably, you mean Category category? Not Question question...

Reply:
You need to specify List<Question> questions in Category entity. According to Java Champion - Vlad Michalcea - bidirectional mapping of 1:M - is the best for JPA/Hibernate.
See the article:
https://vladmihalcea.com/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/